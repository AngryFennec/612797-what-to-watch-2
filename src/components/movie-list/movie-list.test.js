import React from 'react';
import MovieList from './movie-list.jsx';
import renderer from 'react-test-renderer';

const titles = [`Fantastic Beasts`, `Bohemian Rhapsody`, `Macbeth`, `Yet another movie`];

it(`MovieList component renders correctly`, () => {
  const tree = renderer
    .create(<MovieList
      titles = {titles}
      onTitleClick = {jest.fn()}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
