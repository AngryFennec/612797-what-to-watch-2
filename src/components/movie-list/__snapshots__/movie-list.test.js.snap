// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieList component renders correctly 1`] = `
Array [
  " ",
  <article
    className="small-movie-card catalog__movies-card"
    id={0}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        height={175}
        muted={true}
        poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
        src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
        width={200}
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="#"
      >
        Fantastic Beasts: The Crimes of Grindelwald
      </a>
    </h3>
  </article>,
  " ",
]
`;
